exads/examples/prebidJS-client-RTB-InStreamVideo.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
+    <meta charset='UTF-8'>
+    <title>RTB Instream Video Prebid.js</title>
+
+    <!-- Uncomment for development environments -->
+    <!-- <script src='js/prebid.js'></script> -->
+    <!-- Uncomment for production environments -->
+    <script async src='js/prebid.js' onload='onScriptLoaded()'></script>
+
+    <!-- Fluid Player CSS and JS -->
+    <script async src='https://cdn.fluidplayer.com/v3/current/fluidplayer.min.js'></script>
+    <script>
+        // Client Conf start
+        // Set as true for development environments only
+        const isEnabledDebug = false;
+        const exads_endpoint = 'https://your-ad-network.com/rtb.php';
+        const partner = 'rtb_2_4';
+        let zoneId = 5179865;
+        let fid = '12f6c5316c88687e275728a8f24f8787b6c31202';
+
+        // Start GDPR params
```

Comment:
we should now implement DSA here as well

Reply:
Just added in accord to https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/dsa_transparency.md#sample-openrtb-26-bid-request-with-dsa-transparency.

`                dsa: {
                      dsarequired: 3, 
                      pubrender: 0,
                      datatopub: 2
    },`

Check https://github.com/EXADS/Prebid.js/blob/EX-16373/modules/exadsBidAdapter.md